partcad/src/partcad/sketch_factory_enrich.py

```
@@ -19,45 +19,62 @@
             else:
-                source_project = ctx.get_project(self.source_project_name)
-                augmented_config = source_project.get_sketch_config(self.source_sketch_name)
+                self.source_project = self.ctx.get_project(self.source_project_name)
+                if self.source_project is None:
+                    pc_logging.debug("Available projects: %s" % str(sorted(list(self.ctx.projects.keys()))))
+                    raise Exception("Package not found: %s" % self.source_project_name)
+                augmented_config = self.source_project.get_sketch_config(self.source_sketch_name)
             if augmented_config is None:
-                pc_logging.error("Failed to find the sketch to enrich: %s" % self.source_sketch_name)
+                pc_logging.error(
+                    f"Failed to find the sketch to enrich: {self.source_project.name}:{self.source_sketch_name}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)